### PR TITLE
Added k3-fragments plugin

### PR DESCRIPTION
**Pull request by samzzi**

### CODE REVIEW

**Comment by afbora** [comment]
Need "genxbe/plugin-developer.txt" file

**Comment by samzzi** [comment]
@afbora I got a mail/notification from you via github with the msg "Need plugin-developer.txt" (but can't find it here 😊) but that's already there from previous plugins 😬

**Comment by afbora** [comment]
Yes, I realized later and delete the comment :)

**Review by afbora — APPROVED** [review]

File: content/plugins/genxbe/fragments/plugin.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+Repository: https://github.com/genxbe/k3-fragments
+
+----
+
+Category: templating
+
+----
+
+Description: Kirby3 Fragments is a small string/translation helper. You can use fragments in your code 'laravel/blade' style and optionally translate them on site or page level. Additionaly you can pass placeholders to the fragment. A perfect solution for translation of static labels etc... But can also be used to make other 'static' content manageable via the CMS.
```
Comment (lukasbestle) [inline content/plugins/genxbe/fragments/plugin.txt]: The description is quite long for the layout on the plugins page. Suggestion for a shorter version:

```suggestion
Description: Small string/translation helper for fragments 'laravel/blade' style. Supports translating fragments (e.g. static labels) on site or page level and passing placeholders to fragments.
```

**Review by lukasbestle — CHANGES_REQUESTED** [review]

**Comment by lukasbestle** [comment]
@samzzi Thank you very much for your contribution. One comment regarding the description, see above. Could you please also squash your commits for a cleaner commit history?

**Comment by samzzi** [comment]
> @samzzi Thank you very much for your contribution. One comment regarding the description, see above. Could you please also squash your commits for a cleaner commit history?

👍 for the description, thanks for the suggestion! I'll have to search how I can squash commits on an open pull request, I'll fix it once I know how 😬 Thanks for the feedback though!

**Comment by afbora** [comment]
Do you use any program like Github Desktop?

**Comment by lukasbestle** [comment]
I've squash-merged the PR via the GitHub UI. :)

**Comment by samzzi** [comment]
@lukasbestle cheer mate and apologies for the delay in looking into it, i'm swamped at the moment. (I used the only freetime I had to create this plugin, haha ;)) I'll take note for next time though! Thx!

**Pull request merged**